packages/expo-tracking-transparency/plugin/src/withTrackingTransparency.ts

```diff
@@ -0,0 +1,39 @@
+import { ConfigPlugin, createRunOncePlugin } from '@expo/config-plugins';
+
+const pkg = require('expo-tracking-transparency/package.json');
+
+const DEFAULT_NSUserTrackingUsageDescription =
+  'This will allow the app to gather app-related data that can be used for tracking you or your device.';
+
+const withTrackingTransparency: ConfigPlugin<{
+  /**
+   * Sets the iOS `NSUserTrackingUsageDescription` permission message in `Info.plist`.
+   * Passing `false` will skip adding the permission.
```

Comment:
do we do this boolean string stuff in other places? seems like null / omitting it would be better

Reply:
yeah this is the same behavior as the other modules that add `NSUserTrackingUsageDescription`, think it's better to keep the behavior consistent across those modules

I think we want to default to including the permission even if no string is provided in app.json because your app will likely be rejected if the permission isn't included at all